pkg/cosign/env/env.go

```diff
@@ -138,6 +139,12 @@ var (
 			Sensitive:   false,
 			External:    true,
 		},
+		VariableSigstoreTSACertificateFile: {
+			Description: "path to the TSA certificate file used by Sigstore",
```

Comment:
Can you mention this is a concatenated list of PEM-encoded TSA leaf certificate, intermediates and root?